ext/log/build.gradle

```diff
@@ -12,10 +12,12 @@ kotlin {
     sourceSets {
         commonMain.dependencies {
             api Dep.Kotlin.stdlibCommon
+            api Dep.Timber.common
         }
         androidMain {
             dependencies {
                 api Dep.Kotlin.stdlibJvm
+                api Dep.Timber.jdk
```

Comment:
i add Timber dependency in it. but client module needs explicitly Timber dependency.